.github/workflows/kiali-ci.yml

```diff
@@ -7,194 +7,24 @@ on:
   pull_request:
     branches: [master, "^v\\d+\\.\\d+$"]
 
-  # Makes this workflow reusable
-  workflow_call:
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
-
-env:
-  # This workflow can run on either a PR or release. GITHUB_BASE_REF is only
-  # set when this workflow is run against a PR. When run against a PR, we want
-  # to use the branch we are running against: GITHUB_BASE_REF. When run
-  # as a release, we want to use the current branch ref: GITHUB_REF_NAME.
-  TARGET_BRANCH: ${{ github.base_ref || github.ref_name }}
```

Comment:
nit: you can still keep this as an env var and reuse it down below instead of having to redefine it each time like [here](https://github.com/kiali/kiali/pull/4932/files#diff-c9693e0c78da16136b56356cf72c6fcbf3a715cf5fe159b8c7f8dd865c08197cR20).

Reply:
I tried that, but the env vars doesn't play well with reusable jobs. 

I found this that mentions the issue: https://github.community/t/passing-environment-variables-to-reusable-workflow/230456

Reply:
Ah that's annoying that `${{ env.TARGET_BRANCH }}` won't work. You don't need to worry about it then.